Revert "Edge channel of the Gnome SDK only for Risc-V."

nteodosio opened the pull request:
This reverts commit 80c955fe059f038d97a0d7e2bb3aed6c6f359231, as the edge builds of Gnome-46-2404{,-sdk} have been promoted to stable.

seb128 reviewed (approved):
Great!

seb128 reviewed (changes requested):
Or in fact should we just drop it?

nteodosio committed:
Do not use the edge channel of the Gnome SDK.

The edge builds of Gnome-46-2404{,-sdk} have been promoted to stable
so we no longer need this work-around.

nteodosio commented:
Oops, thanks for catching it, now fixed.

seb128 reviewed (approved):
Thanks
